ODIN_II/regression_test/benchmark/task/preprocessor/simulation_result.json

```diff
@@ -201,6 +201,87 @@
         "Estimated LUTs": 1,
         "Total Node": 1
     },
+    "preprocessor/elsif_both_defined/no_arch": {
```

Comment:
These test arent mine and they havent had results generated previously. Was there a reason for this or just someone forgot? Should I leave them out

Reply:
No, it seems there was some code changes that made the preprocessor work?? (Weird) and now since it synthesised it simulates, I'll double check